scripts/item.py

```diff
@@ -1,38 +1,24 @@
 import base64
 import json
-import os
 import sys
 
-import boto3
 import requests
-from utils import DATA_PATH, MWAA_NAME, data_files
-
-dag_name = "veda_discover"
-airflow_client = boto3.client("mwaa")
-print(MWAA_NAME)
-mwaa_cli_token = airflow_client.create_cli_token(Name=MWAA_NAME)
-mwaa_auth_token = "Bearer " + mwaa_cli_token["CliToken"]
-mwaa_webserver_hostname = "https://{0}/aws_mwaa/cli".format(
-    mwaa_cli_token["WebServerHostname"]
-)
-
-
-items_path = os.path.join(DATA_PATH, "items")
+from utils import get_items, get_mwaa_cli_token
 
 
 def insert_items(files):
+    mwaa_cli_token = get_mwaa_cli_token()
+    mwaa_auth_token = "Bearer " + mwaa_cli_token["CliToken"]
     print("Inserting items:")
     for filename in files:
         print(filename)
         events = json.load(open(filename))
         if type(events) != list:
             events = [events]
         for event in events:
-            raw_data = "dags trigger {0} --conf '{1}'".format(
-                dag_name, json.dumps(event)
-            )
+            raw_data = f"dags trigger veda_discover --conf '{json.dumps(event)}'"
```

Comment:
weren't we talking about using the [api](https://4e76fa52-1de6-41a5-b364-691fb1cc69f4.c18.us-west-2.airflow.amazonaws.com/api/v1/ui/#/DAGRun/post_dag_run) rather than the cli?

Reply:
actually, just saw your reply to the same question on another PR. nvm.